static/component/catalog_table.html

```diff
@@ -0,0 +1,108 @@
+<style type='text/css'>
+  /* Set search bar's height to 64px. Otherwise, the UI breaks,
+    it is a bug in materlize-css.*/
+  input[type='search'] {
+    height: 64px !important;
+  }
+  /* Sticky column interface / API column. */
+  .colhead {
+    position: absolute;
+    left: auto;
+    top: auto;
+    background-color: rgba(255, 255, 255, 0.5);
+  }
+</style>
+<div class='card-panel'>
+  <nav class='blue lighten-3'>
+    <div class='nav-wrapper'>
+      <ul class='left'>
+        <!-- Pagination -->
+        <li class='waves-effect'
+          ng-class='{disabled: currentPage === 0}'
+          ng-click='setPage(currentPage-1)'
+          ng-disabled='currentPage === 0'>
+          <i class='material-icons'>chevron_left</i>
+        </li>
+        <li class='waves-effect'
+          ng-class='{active: n === currentPage}'
+          ng-repeat='n in range(pageLength, currentPage)'>
```

Comment:
style nit: move the ng-repeat above ng-class - it helps make the repeat more obvious.